### ISSUE TITLE
Add NWC connector for lightning wallets

### ISSUE THREAD

**Issue by itstomekk** [BlueWallet/BlueWallet#6558]
[Nostr Wallet Connect (NWC) ](https://nwc.dev/)is an open protocol to connect apps with lightning nodes/wallets.

Currently Blue uses LND and lightning support makes this app people's favorite wallet.

Many users might actually prefer to link Blue to their Umbrel, Start9 or Alby via NWC, as it gives more security with customized budget and expiry times.

**Comment by sjorsvanheuveln** [comment]
I want this feature too, so I can connect Blue Wallet with Damus.

**Comment by itstomekk** [comment]
You couldn't, you still need some lightning funding source (a node that Blue will give wallet interface to).

This request is to make Blue use NWC as a protocol to connect to nodes.

**Comment by seedcard** [comment]
Would also like to add my support to this suggestion. I beieve that the work that is being done with the development of Alby Hub makes it a top contender for pushing forward the  growth in self custodial lightning nodes.  Blue wallet  is a proven product and one the few that can generate both lightning and mainet wallets in the same app. I hope that the devs will see the benefit of keeping up to date and use NWC to connect.

**Comment by stackingsaunter** [comment]
Would be awesome to have it in Blue!

**Comment by supertestnet** [comment]
Yet another NWC fan here chiming in to say it would be awesome if blue wallet supported it

**Comment by MoritzKa** [comment]
Yes, please add NWC!

**Comment by Emcleswern** [comment]
Thumbs up

**Comment by pursuingVirtue** [comment]
Yep. Count me in. LNDhub's requirement for a clearnet address adds technical complexity and can compromise security and privacy. Adding support for NWC would be a fantastic feature for BlueWallet, simplifying setup and enhancing user privacy and security.

**Comment by dadofsambonzuki** [comment]
+1

**Comment by beerye-nostr** [comment]
Just adding +1 for blue wallet adding nwc connection 🤙
